cpp/src/plasma/malloc.cc

```diff
@@ -40,7 +44,12 @@ int fake_munmap(void*, int64_t);
 #define USE_DL_PREFIX
 #define HAVE_MORECORE 0
 #define DEFAULT_MMAP_THRESHOLD MAX_SIZE_T
-#define DEFAULT_GRANULARITY ((size_t)128U * 1024U)
+
+#ifndef ARROW_PLASMA_HUGEPAGES
+#define DEFAULT_GRANULARITY ((size_t)128U * 1024U)  // 128KB
```

Comment:
Would making this a configuration option that can be changed ulon initialization of Plasma be expensive?